src/containers/Login.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from "react";
+import { Button, FormGroup, FormControl, ControlLabel } from "react-bootstrap";
+import "./Login.css";
+
+export default class Login extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      email: "",
+      password: ""
+    };
+  }
+
+  validateForm() {
+    return this.state.email.length > 0 && this.state.password.length > 0;
```

Comment:
I'd suggest that you use Formik, as a very popular form library, it will provide things like "isTouched", Field and Form tags out of the box and is widely used in the ecosystem.

Reply:
I'll heed your words